Allow Movable Insulation to Support Equivalent Layer Glazing

RKStrand opened the pull request:
Pull request overview
---------------------
Resolves #7385. A recent fix included a unit test that checked to see if movable insulation works with equivalent layer glass.  The unit test is successful and it appears from the code that this was considered an appropriate input for movable insulation.  Unfortunately, there was another place in the code that prevented that option.  This has now been fixed and the documentation has been clarified.

### Work Checklist
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [x] Title of PR should be user-synopsis style (clearly understandable in a standalone changelog context)
 - [x] At least one of the following appropriate labels must be added to this PR to be consumed into the changelog:
   - Defect: This pull request repairs a github defect issue #7385 

### Review Checklist
This will not be exhaustively relevant to every PR.
 - [ ] Functional code review (it has to work!)
 - [ ] If defect, results of running current develop vs this branch should exhibit the fix
 - [ ] CI status: all green or justified
 - [ ] Performance: CI Linux results include performance check
 - [ ] Unit Test(s)
 - C++ checks:
   - [ ] Argument types
   - [ ] If any virtual classes, ensure virtual destructor included, other things
 - IDD changes:
   - [ ] Verify naming conventions and styles, memos and notes and defaults
   - [ ] Open windows IDF Editor with modified IDD to check for errors
   - [ ] If transition, add to input rules file for interfaces
   - [ ] If transition, add transition source
   - [ ] If transition, update idfs
 - [ ] If new idf included, locally check the err file and other outputs
 - [ ] Required documentation updates?
 - [ ] ExpandObjects changes?
 - [ ] If output changes, including tabular output structure, add to output rules file for interfaces

RKStrand committed:
Solution allows GlassEquivalentLayer to be used

This change allows GlassEquivalentLayer materials to be used to define
Movable Insulation.  The calculation code for movable insulation
previously allowed this, but the get routine blocked this material from
being used for this.  This corrects that and adds a new required
thermal resistance and a default value that is needed by the simulation.

RKStrand committed:
Addition of a unit test and doc changes

Added a unit test to verify that the new input field is being
interpreted properly (a default case).  Also added documentation to
give information on the new field.

RKStrand committed:
Correction of idd

Warning message revealed that the units for the new field were in
incorrect format.

Nigusse commented:
Tested the use of Glazing Equivalent Layer material with movable insulation object. It looks the model is working as expected. Below is a sample output, the outside wall surface temperature is impacted by the equivalent layer glazing used as "movable insulation layer" on the outside..

![MovableInsulationImpact](https://user-images.githubusercontent.com/5805126/62794982-90020d80-baa3-11e9-8f66-0d23653814a7.png)

Nigusse commented on doc/input-output-reference/src/overview/group-surface-construction-elements.tex:
```diff
@@ -3386,6 +3386,10 @@ \subsubsection{Inputs}\label{inputs-30-001}
 
+\paragraph{Field: Thermal Resistance}\label{field-thermal-resistance-equivglazing}
+
+This field is used to enter the thermal resistance (R-value) of the material layer. Units for this parameter are (m\(^{2}\)-K)/W. Thermal resistance must be greater than zero.  The default value is 0.158 which is roughly equivalent to a single layer of 1/4" glass.
```
@RKStrand should we add a note that field is used with movable insulation only?

Nigusse commented on idd/Energy+.idd.in:
```diff
@@ -6960,13 +6960,20 @@ WindowMaterial:Glazing:EquivalentLayer,
        \minimum> 0.0
        \maximum< 1.0
        \default 0.84
+  N26; \field Thermal Resistance
```
@RKStrand should we add the same note here as well.

Nigusse reviewed (commented):
@RKStrand if you address the two comments, this fix is ready to merge.

RKStrand committed:
Merge branch 'develop' into 7385-MovableInsulationSupportEquivalentWindow

RKStrand committed:
Merge branch 'develop' into 7385-MovableInsulationSupportEquivalentWindow

RKStrand committed:
Corrections in response to code review

A couple of minor changes prompted by the review for this defect are
included in this commit.

RKStrand commented:
@Nigusse Thanks for your review, Bereket!  I have made changes per your suggestions and have committed those changes.  Assuming that the CI runs come back ok (which hopefully it will) and you are okay with these edits, this should be ready to merge.

Nigusse committed:
more InputOutput reference documentation update

Nigusse commented:
I have reviewed this fix and approved for merging. @Myoldmopar would you like to look at it before I merge?

Myoldmopar commented:
@Nigusse this looks good to me, feel free to merge.
